experimental/dds/tree2/src/feature-libraries/editable-tree/editableField.ts

```diff
@@ -61,8 +61,19 @@ export function makeField(
 	fieldSchema: FieldStoredSchema,
 	cursor: ITreeSubscriptionCursor,
 ): EditableField {
-	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor);
-	return adaptWithProxy(targetSequence, fieldProxyHandler);
+	const fieldAnchor = cursor.buildFieldAnchor();
+
+	const targetSequence = new FieldProxyTarget(context, fieldSchema, cursor, fieldAnchor);
+	const output = adaptWithProxy(targetSequence, fieldProxyHandler);
+	if (fieldAnchor.parent !== undefined) {
+		const anchorNode =
+			context.forest.anchors.locate(fieldAnchor.parent) ??
+			fail("cursor should point to a node that is not the root of the AnchorSet");
```

Comment:
this comment is not accurate in this case. Maybe this would be better:
```suggestion
			fail("parent anchor node should always exist since field is under a node");
```

Reply:
corrected fail message